atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -160,6 +160,10 @@ def get_cachito_session(workflow):
     return CachitoAPI(config['api_url'], **api_kwargs)
 
 
+def get_allow_multiple_remote_sources(workflow):
+    return get_value(workflow, 'allow_multiple_remote_sources', False)
```

Comment:
Should we have it enabled by default?

Reply:
better not yet, maybe in the future

Reply:
in future it will be the only way, so the whole option will be deprecated